openupgrade_scripts/scripts/point_of_sale/15.0.1.0.1/pre-migration.py

```diff
@@ -3,8 +3,30 @@
 from openupgradelib import openupgrade
 
 
+def merge_pos_order_return_module(env):
+    if openupgrade.column_exist(env.cr, "pos_order_line", "returned_order_id"):
```

Comment:
Wrong table :point_down: 
```suggestion
    if openupgrade.column_exist(env.cr, "pos_order", "returned_order_id"):
```